src/pimp_mixer_arm.S

```diff
@@ -298,9 +298,10 @@ pimp_mixer_mix_samples:
 	ldr r4, =0x4000208 @ load address of REG_IME
 	ldr r5, .ime_store @ stash for later
```

Comment:
Similarly, I think this comment should have read "restore value"

Reply:
This was just a note to myself, though. I'm not asking you to fix the comment ;)